apps/graphql/src/apps/itinerary/Itinerary.js

```diff
@@ -16,7 +16,7 @@ type Passengers = {|
 |};
 
 type SearchLocation = {|
-  +ids: string[],
+  ids: string[],
```

Comment:
Why you removed read-only ?

Reply:
Because we are modifying it by casting it `fromGlobalID` and flow complained. Or maybe I can clone to object to avoid that.

Reply:
Placed it back!

Reply:
As I can see you use the fromGlobalID with the `map` and `map` return new Array so it should not be modified.